sklearn/ensemble/_hist_gradient_boosting/grower.py

```diff
@@ -5,6 +5,9 @@
 the gradients and hessians of the training data.
 """
 
+# Authors: The scikit-learn developers
+# SPDX-License-Identifier: BSD-3-Clause
+
 # Author: Nicolas Hug
```

Comment:
Do we keep lines like this? I don't remember if we discussed/decided something, my knee jerk reaction while looking at this PR is "we should remove it".

Reply:
My understanding is that we'll remove them all at some point. It can be done in follow-up PRs though

Reply:
Another PR was supposed to remove these, this was missed.